api/src/main/resources/hibernate.cfg.xml

```diff
@@ -41,7 +41,6 @@
 		<mapping resource="org/openmrs/api/db/hibernate/ConceptReferenceTerm.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/ConceptMapType.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/ConceptReferenceTermMap.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Drug.hbm.xml" />
```

Comment:
You deleted this line with `*.hbm.xml` but is there reason why the actual file is not deleted?

Reply:
No, I will delete it, I was waiting for your review